### ISSUE TITLE
Crash in torchvision tutorial sample code

### ISSUE THREAD

**Issue by aCuria** [pytorch/pytorch#33535]
## 🐛 Bug

Crash when running "tv-training-code.py"
https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html

![image](https://user-images.githubusercontent.com/35476457/74906690-b4dd2300-53ec-11ea-8d15-984a97bdf8a6.png)

## To Reproduce

1. Run tv-training-code.py with python 3.7, pycocotools 2.00, vs2019
2. Crash after first epoch

Epoch: [0]  [ 0/60]  eta: 0:01:59  lr: 0.000090  loss: 3.6203 (3.6203)  loss_classifier: 0.6837 (0.6837)  loss_box_reg: 0.1773 (0.1773)  loss_mask: 2.7259 (2.7259)  loss_objectness: 0.0237 (0.0237)  loss_rpn_box_reg: 0.0098 (0.0098)  time: 1.9867  data: 0.7440  max mem: 2139
Epoch: [0]  [10/60]  eta: 0:00:27  lr: 0.000936  loss: 1.8186 (2.3457)  loss_classifier: 0.4372 (0.4416)  loss_box_reg: 0.1773 (0.1920)  loss_mask: 1.2114 (1.6839)  loss_objectness: 0.0110 (0.0167)  loss_rpn_box_reg: 0.0101 (0.0115)  time: 0.5418  data: 0.0683  max mem: 3076
Epoch: [0]  [20/60]  eta: 0:00:18  lr: 0.001783  loss: 0.7622 (1.5273)  loss_classifier: 0.1937 (0.3105)  loss_box_reg: 0.1568 (0.1637)  loss_mask: 0.3449 (1.0187)  loss_objectness: 0.0140 (0.0205)  loss_rpn_box_reg: 0.0111 (0.0139)  time: 0.3805  data: 0.0006  max mem: 3076
Epoch: [0]  [30/60]  eta: 0:00:12  lr: 0.002629  loss: 0.4866 (1.1928)  loss_classifier: 0.1035 (0.2428)  loss_box_reg: 0.1189 (0.1560)  loss_mask: 0.2358 (0.7609)  loss_objectness: 0.0163 (0.0183)  loss_rpn_box_reg: 0.0122 (0.0149)  time: 0.3697  data: 0.0007  max mem: 3076
Epoch: [0]  [40/60]  eta: 0:00:08  lr: 0.003476  loss: 0.4439 (1.0093)  loss_classifier: 0.0733 (0.2025)  loss_box_reg: 0.1218 (0.1555)  loss_mask: 0.1967 (0.6204)  loss_objectness: 0.0063 (0.0153)  loss_rpn_box_reg: 0.0147 (0.0156)  time: 0.3830  data: 0.0007  max mem: 3076
Epoch: [0]  [50/60]  eta: 0:00:04  lr: 0.004323  loss: 0.4103 (0.8831)  loss_classifier: 0.0628 (0.1729)  loss_box_reg: 0.1218 (0.1463)  loss_mask: 0.1835 (0.5359)  loss_objectness: 0.0041 (0.0129)  loss_rpn_box_reg: 0.0147 (0.0152)  time: 0.3934  data: 0.0006  max mem: 3134
Epoch: [0]  [59/60]  eta: 0:00:00  lr: 0.005000  loss: 0.3786 (0.8003)  loss_classifier: 0.0506 (0.1553)  loss_box_reg: 0.0915 (0.1357)  loss_mask: 0.1835 (0.4829)  loss_objectness: 0.0015 (0.0114)  loss_rpn_box_reg: 0.0141 (0.0149)  time: 0.3875  data: 0.0006  max mem: 3134
Epoch: [0] Total time: 0:00:24 (0.4126 s / it)
creating index...
index created!
## Expected behavior

Crash should not happen in sample code

## Environment

python .\collect_env.py
Collecting environment information...
PyTorch version: 1.4.0
Is debug build: No
CUDA used to build PyTorch: 10.1

OS: Microsoft Windows 10 Enterprise
GCC version: Could not collect
CMake version: version 3.16.0-rc1

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: 10.1.105
GPU models and configuration:
GPU 0: GeForce GTX 1080 Ti
GPU 1: GeForce GTX 1080 Ti

Nvidia driver version: 441.99
cuDNN version: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\bin\cudnn64_7.dll

Versions of relevant libraries:
[pip3] numpy==1.18.1
[pip3] torch==1.4.0
[pip3] torchvision==0.5.0
[conda] Could not collect

## Additional context

cc @ezyang @gchanan @zou3519 @fmassa

**Comment by ptrblck** [comment]
@aCuria Do you see any error message or is your code hanging after the last print statement?

**Comment by aCuria** [comment]
> @aCuria Do you see any error message or is your code hanging after the last print statement?

There is no further error or output in the console window other than what is shown in the image.

**Comment by peterjc123** [comment]
cc @fmassa Looks like the crash happened in torchvision code

**Comment by aCuria** [comment]
I tried to step into the COCOeval() code but it doesn't really let me go deep enough into the source to find the problem. 

Any idea if an older/newer version of torchvision should work?

**Comment by peterjc123** [comment]
> I tried to step into the COCOeval() code but it doesn't really let me go deep enough into the source to find the problem.

For debugging purposes, you'll need to build a RelWithDebInfo version of torchvision. 

> Any idea if an older/newer version of torchvision should work?

You may try that. But I guess you'll need to downgrade PyTorch as well.

**Comment by albanD** [comment]
High priority until we get a repro as a crash in a tutorial is not good.

**Comment by peterbell10** [comment]
I was able to reproduce, it appears to be a bug in `pycocotools`. The stack trace lead me to cocodataset/cocoapi#354 which does fix the issue, although it isn't in any released version yet.

In the mean time, I can confirm that downgrading to `numpy==1.17` is a valid workaround.

Closing since this is already fixed in `pycocotools`, pending a new release.
